templates/py-simple/main.py

```diff
@@ -1,3 +1,3 @@
 async def run(context, input):
-   async for id in input:
-      yield f"id:{id}"
+	async for id in input:
+		yield f"id:{id}"
```

Comment:
does string conversion on returned value is really a simple template? Shouldn't that be just `yield id`